handbook/company/ceo.md

```diff
@@ -268,8 +268,25 @@ I held on as long as I could.  But due to volume, in late 2022, I made the decis
  - If you mention me in a top-level channel message, I'll see and read it in 1 business day.
  - If you send me a direct message, I'll see and read that ASAP.
 Keep in mind I am often in meetings all day, and may not be able to reply promptly.
-When in doubt, you can look at my calendar and join whatever meeting I'm in.  If none of that works, and there is an emergency where you need my immediate attention, get help from Zach Wasserman.
-Thank you so much!" 🙇</blockquote> 
+Thank you so much!" 🙇
+
+
+## Slack channels
+
+| Channel | Purpose | Standard Operating Procedure | DRI |
+| ------- | ------- | ---------------------------- | --- |
+| [g-ceo](https://fleetdm.slack.com/archives/C05K87USYUV)  
+| Open line of communication for Fleeties to make requests of the 🦿 [CEO support team](https://fleetdm.com/handbook/company/ceo#team) 
+| Reach out to the AttC with requests or to add an item on the  | AttC |
+| [help-being-ceo](https://fleetdm.slack.com/archives/C03U703J0G5) | Private [hallway](https://about.gitlab.com/handbook/leadership/1-1/suggested-agenda-format/#hallway) for the CEO to drop tasks and personal scheduling information | - Mark items with 👀 (emoji reaction) to indicate that you are working on the task. - Mark requests with ✅ (emoji reaction) to indicate that you completed or filed away the task | AttC |
+
+     
+## Rituals
+(Who will...)
+
+<rituals :rituals-table="ritualsTables.company"></rituals>
```

Comment:
~~Vision is change this to `<rituals path="./ceo-rituals.yml"></rituals>`  (we can come back to that)~~

Nope, I'm wrong.

Instead:

```
<rituals :rituals="rituals['handbook/company/ceo-rituals.yml']"></rituals>
```

Reply:
<img width="447" alt="image" src="https://github.com/fleetdm/fleet/assets/618009/019b4a37-60b5-4e86-8851-2550b73dd905">